sw/otbn/crypto/tests/mlkem/mlkem_base_codesize.s

```diff
@@ -0,0 +1,638 @@
+/* Copyright "Towards ML-KEM & ML-DSA on OpenTitan" Authors */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Modified by Ruben Niederhagen and Hoang Nguyen Hien Pham - authors of */
+/* "Improving ML-KEM & ML-DSA on OpenTitan - Efficient Multiplication Vector Instructions for OTBN" */
+/* (https://eprint.iacr.org/2025/2028) */
+/* Copyright Ruben Niederhagen and Hoang Nguyen Hien Pham. */
+
+
+/*
+ * Test to obtain ML-KEM code size
```

Comment:
I think we probably don't need to import this file or `mlkem_codesize.s` -- they seem to be just for collecting code size measurements for the paper.